app/jobs/zipmaker_job.rb

```diff
@@ -9,6 +9,11 @@ class ZipmakerJob < ApplicationJob
   # @param [String] druid
   # @param [Integer] version
   def perform(druid, version)
+    binary_path = Moab::StorageServices.object_version_path(druid, version)
+    zip_path = "#{Settings.zip_storage}/transfers/#{druid}#{format('.v%04d.zip', version)}"
```

Comment:
I thought we would still use druid-tree subdirs for zip storage for performance reasons.  I don't think it should be necessary to do `#{Settings.zip_storage}/transfers`.  If a `transfers` (or any other) subdirectory is needed, it should be part of `Settings.zip_storage`.  This logic has to match (or ideally reuse) the zip-locating logic in `DruidVersionZip` or our pieces wont fit together.

When the zip exists we should `FileUtils.touch` the file to prevent it being cleaned out by whatever cache-maintenance code we write later.

Reply:
I would say dealing with zip files needing to be cleaned out should be a separate ticket;  o.w. I agree with @atz.